File: modle_backend/src/main/java/com/modle/domain/jobposting/entity/Category.java
package com.modle.domain.jobposting.entity;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum Category {
    HAIR("헤어"),
    MAKEUP("메이크업"),
    CLOTHING("의류"),
    FITTING("피팅"),
    HAND("손"),
    FOOD("푸드"),
    PRODUCT("제품"),
    ETC("기타");

    private final String displayName;
}

File: modle_frontend/src/lib/constants/category.ts
export const CATEGORY_OPTIONS = [
  { value: "HAIR", label: "헤어" },
  { value: "MAKEUP", label: "메이크업" },
  { value: "CLOTHING", label: "의류" },
  { value: "FITTING", label: "피팅" },
  { value: "HAND", label: "핸드" },
  { value: "FOOD", label: "음식" },
  { value: "PRODUCT", label: "제품" },
  { value: "ETC", label: "기타" },
] as const;

// 카테고리 enum 값을 한글 라벨로 변환 (매칭 없으면 원래 값 반환)
export function getCategoryLabel(value?: string | null): string {
  return CATEGORY_OPTIONS.find((o) => o.value === value)?.label ?? value ?? "";
}
